build.sbt

```diff
@@ -33,7 +34,7 @@ val commonSettings = Seq(
     Libraries.scalaCheck % Test
   ),
   resolvers += "Apache public" at "https://repository.apache.org/content/groups/public/",
-  scalacOptions ++= Seq("-Xmax-classfile-name", "80"),
+//  scalacOptions ++= Seq("-Xmax-classfile-name", "80"),
```

Comment:
If this doesn't work on 2.13 we should pattern match on the version and set the proper flags.